project_national/tests/test.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require_relative '../../resources/hpxml-measures/HPXMLtoOpenStudio/resources/minitest_helper'
 require_relative '../../test/integrity_checks'
 require 'minitest/autorun'
```

Comment:
It's not a big deal, but you can also remove the minitest/autorun lines if you want, since it's part of minitest_helper.